bin/cli.js

```diff
@@ -55,25 +55,19 @@ const cliFlags = yargs
     'default': false
   })
   .epilogue('For more Lighthouse CLI options see https://github.com/GoogleChrome/lighthouse/#lighthouse-cli-options')
-  .check(argv => {
-    //test possible url locations, first from cmd line then from config file
-    if (argv._.length === 0 && (argv.config === undefined || !argv.config.url)) {
-      throw new Error(getMessageWithPrefix('ERROR', 'NO_URL'));
-    }
-    return true;
-  }).argv;
-
+  .argv;
 
 const config = cliFlags.config ? getConfigFromFile(cliFlags.config) : {};
 
-// For the config flags and URL, we'll allow CLI to override what's set in the config Object
-
 //Merge options from all sources. Order indicates precedence (last one wins)
-let options = Object.assign({}, config, {flags: cliFlags});
+let options = Object.assign({}, {flags: cliFlags}, config);
```

Comment:
Let config from file replace cli for now. Previous one will rewrite file config by default `cliFlags` values from yargs.